css/hard-mode-toggle.css

```diff
@@ -0,0 +1,52 @@
+.hide {
+    display: none;
+}
+
+.toggle-container p {
+    text-align: center;
+    color: orange;
+    grid-row: inherit;
+    margin: 10px;
+}
+
+.hard-mode-toggle, .toggle-fill {
+    border-radius: 20%;
+    width: 50px;
+    margin: 0 auto;
+    height: 5px;
+    background-color: white;
+    position: relative;
+}
+
+.toggle-fill {

```

Comment:
This element does not display correctly on mobile view and shows a pixel or two of the white background above it.